### DISCUSSION TITLE
Did the Addon still work?

### DISCUSSION THREAD

**Discussion by Burn-In** [maxbec/hassio-diyHue#23] [Q&A]
Have anyone this Addon working with an updated setup? Since some month the Addon have been stopped working on my Raspi. (Bridge didn't find lights, "Aborted (core dumped)", Ambilight-TV didn't find bridge) Multiple HA updates didn't fix this. I did also several HA clean installs. So I'm asking me, has nobody the same issues? What do you other then I? I wished I had never touched my running system.

**Comment by chillkiller** [comment]
near same issue here... Ha image on a RPI4/4G, DeConz addon (Conbee2 Stick), DIYHue addon. tried to couple deconz with DIYHue, doesnt find Deconz and ask of pairing switch pushed, tried to couple DIYHue with ofifcial Philips Hue App makes an error on the app, tried to couple DIYHue with Philips TV, TV doesnt find DIYHue...

**Comment by foxy82** [comment]
Interesting I've had the app working and the Phillips hue TV. I don't connect mine to deconz instead I use the home assistant integration in diyhue to expose the deconz lights. 

Have a look here and see if your setup is similar: https://community.home-assistant.io/t/guide-using-the-hue-app-with-home-assistant-via-diyhue/296684

**Reply by chillkiller** [reply]
didnt want to integrate all my lights and switches to diyhue by switching from hue bridge to deconz step by step... but before I'm sure that all works, I will not break my complete automations

**Reply by foxy82** [reply]
I think I've seen the core dump before if there is a port clash but I can't properly remember now. I think it might be clashes on the 443 port.

**Reply by foxy82** [reply]
> didnt want to integrate all my lights and switches to diyhue by switching from hue bridge to deconz step by step... but before I'm sure that all works, I will not break my complete automations

From what I understand you can connect the diyhue app directly to deconz. At least there is mention of that on the deconz GitHub have you tried that it sounds like that would mean you don't need diyhue?

**Reply by foxy82** [reply]
> didnt want to integrate all my lights and switches to diyhue by switching from hue bridge to deconz step by step... but before I'm sure that all works, I will not break my complete automations

You can connect the hue app to two bridges (although not at the same time). If you are pulling hue bulbs and deconz bulbs into home assistant then you can use the diyhue/home assistant integration to expose all of them in the hue app. That way you could make it fairly painless at the hue app end (but would require work in home assistant as you migrate).

**Reply by chillkiller** [reply]
I wanted to use DIYhue for other Lights and sensors like my esphome sensors and switches ;)
Also hoped to smash the hue bridge in the trash if I get the philips tv work without the old bridge (last generation)

**Reply by foxy82** [reply]
I can see some errors for the diyhue / home assistant integration with the latest home assistant. Although that won't be your problem if you don't use that part of diyhue

**Reply by foxy82** [reply]
> I wanted to use DIYhue for other Lights and sensors like my esphome sensors and switches ;)
> Also hoped to smash the hue bridge in the trash if I get the philips tv work without the old bridge (last generation)

I've had the TV working but if you do that you really are risking Phillips hue breaking compatibility and relying on others to fix it. 

The biggest issue you are likely to face getting diyhue working on the home assistant box is that it needs port 80 and 443 so you need to make sure nothing is using them. I personally use those for the home assistant front end hence the setup instructions above

**Reply by Burn-In** [reply]
For my setup, there is no issue with the ports. diyHue is reachable under 80 and 443. Also I have tested a clean HA install and the Addon only. Btw, I use the HA OS image to install.

**Comment by Burn-In** [comment]
> Have anyone this Addon working with an updated setup? Since some month the Addon have been stopped working on my Raspi. (Bridge didn't find lights, "Aborted (core dumped)", Ambilight-TV didn't find bridge) Multiple HA updates didn't fix this. I did also several HA clean installs. So I'm asking me, has nobody the same issues? What do you other then I? I wished I had never touched my running system.

**Comment by Burn-In** [comment]
I could change one thing. After I uninstalled all other add-ons, including Deconz, i could delimit this issue to the "Check Home Assistant configuration" addon. If it doesn't run, "core dump" disappears.
But, the bridge still find the diyHue lights, however the webserver of the lights are reachable. What could I check further, any suggestions?
Edit: Also the Deconz addon forces a "core dump". :(

**Reply by alanpi** [reply]
did you tryout https://github.com/hass-emulated-hue/core i tried diyhue today but i cant run it. with emulated hue i can connect it to hueessentials

**Reply by Burn-In** [reply]
Wow, what a great tip, thanks.
Installed in seconds without need to config. Hue Essentials finds Bridge automatically. Even the Ambilight of my TV appears as a light in Hue Essentials.
But, unfortunately, didn't find diyHue-ESP-light. Have you tried it?
